zetta_utils/training/lightning/train.py

```diff
@@ -364,11 +354,8 @@ def _lightning_train_remote(
         restart_policy="Never",
         # with multinode ddp we need a node on standard pool for an IP
         # that remains the same for the duration of training
-        # these pools must have `master-pool=true` taint
-        # not ncessary for single node ddp so it can be scheduled on preemptibles
-        tolerations=_get_tolerations(
-            "worker"
-        ),  # _get_tolerations(role="master" if num_nodes > 1 else "worker"),
+        node_selector={"cloud.google.com/gke-provisioning": "standard"},
```

Comment:
this should switch to the user selected prov model when num_nodes == 1, right?

Reply:
done